virtualizarr/tests/test_xarray.py

```diff
@@ -313,14 +310,13 @@ def test_mixture_of_manifestarrays_and_numpy_arrays(
         assert isinstance(renamed_vds["lat"].data, np.ndarray)
 
 
-@requires_kerchunk
 def test_nbytes(simple_netcdf4):
     vds = open_virtual_dataset(simple_netcdf4)
-    assert vds.virtualize.nbytes == 32
-    assert vds.nbytes == 48
+    assert vds.virtualize.nbytes == 88
+    assert vds.nbytes == 104
 
     vds = open_virtual_dataset(simple_netcdf4, loadable_variables=["foo"])
-    assert vds.virtualize.nbytes == 48
+    assert vds.virtualize.nbytes == 104
```

Comment:
I guess I am not concerned that the `nbytes` is different now?

Reply:
That seems weird... I would have expected them to be the same.

Reply:
I think this has to do with coordinates being populated for dimensions that are supposed to be "without coordinates" see https://github.com/zarr-developers/VirtualiZarr/pull/395/files#r1934273183.